static/app/components/events/featureFlags/featureFlagOnboardingSidebar.tsx

```diff
@@ -0,0 +1,399 @@
+    'mode',
+    defaultTab
+  );
+
+  const currentPlatform = currentProject.platform
+    ? platforms.find(p => p.id === currentProject.platform) ?? otherPlatform
+    : otherPlatform;
+
+  const {
+    docs,
+    dsn,
+    isLoading: isProjKeysLoading,
+    projectKeyId,
+  } = useLoadGettingStarted({
+    platform: currentPlatform,
+    projSlug: currentProject.slug,
+    orgSlug: organization.slug,
+    productType: 'featureFlags',
+  });
+
+  const radioButtons = (
+    <Header>
+      <StyledRadioGroup
+        label="mode"
+        choices={[
+          [
+            'openFeature',
+            <PlatformSelect key="platform-select">
+              {tct(
+                'I use the [link:OpenFeature] SDK using a provider from [providerSelect]',
+                {
+                  providerSelect: (
+                    <CompactSelect
+                      triggerLabel={openFeatureProvider.label}
+                      value={openFeatureProvider.value}
+                      onChange={setOpenFeatureProvider}
+                      options={openFeatureProviderOptions}
+                      position="bottom-end"
+                      key={openFeatureProvider.textValue}
+                      disabled={setupMode() === 'other'}
+                    />
+                  ),
+                  link: <ExternalLink href="" />, // TODO: link
+                }
+              )}
+            </PlatformSelect>,
+          ],
+          [
+            'other',
+            <PlatformSelect key="platform-select">
+              {tct('I use an SDK from [providerSelect]', {
+                providerSelect: (
+                  <CompactSelect
+                    triggerLabel={sdkProvider.label}
+                    value={sdkProvider.value}
+                    onChange={setsdkProvider}
+                    options={sdkProviderOptions}
+                    position="bottom-end"
+                    key={sdkProvider.textValue}
+                    disabled={setupMode() === 'openFeature'}
+                  />
+                ),
+              })}
+            </PlatformSelect>,
+          ],
+        ]}
+        value={setupMode()}
+        onChange={setSetupMode}
+      />
+    </Header>
+  );
+
+  if (isProjKeysLoading) {
+    return (
+      <Fragment>
+        {radioButtons}
+        <LoadingIndicator />
+      </Fragment>
+    );
+  }
+
+  const doesNotSupportFeatureFlags = currentProject.platform
+    ? !featureFlagOnboardingPlatforms.includes(currentProject.platform)
+    : true;
+
+  if (doesNotSupportFeatureFlags) {
+    return (
+      <Fragment>
+        <div>
+          {tct(
+            'Feature Flags isn’t available for your [platform] project. It is currently only available for Python and JavaScript projects.',
+            {platform: currentPlatform?.name || currentProject.slug}
+          )}
+        </div>
+        <div>
+          <LinkButton
+            size="sm"
+            href="" // TODO: docs link
```

Comment:
wasn't sure what link to use here, i don't think we have a general feature flag page in docs yet

Reply:
use link created in https://github.com/getsentry/sentry-docs/pull/11879